sdcm/nemesis.py

```diff
@@ -443,7 +443,15 @@ def _terminate_cluster_node(self, node):
         self.cluster.terminate_node(node)
         self.monitoring_set.reconfigure_scylla_monitoring()
 
-    def disrupt_nodetool_decommission(self, add_node=True):
+    def disrupt_nodetool_decommission(self, add_node=True, is_seed_node=False):
```

Comment:
create a new disrupt function, since some of the ChaosMonkey logic is base only on function with no ability to call with different parameters

Reply:
Fixed

Reply:
Fixed

Reply:
done